core/server/data/migrations/versions/2.13/1-remove-empty-strings.js

```diff
@@ -0,0 +1,74 @@
+const Promise = require('bluebird');
+const common = require('../../../../lib/common');
+const schema = require('../../../schema');
+
+/*
+ * [{
+ *   tableName: 'posts',
+ *   columns: ['custom_excerpt', 'description', 'etc...']
+ * }]
+ * */
+const tablesToUpdate = Object.keys(schema.tables).reduce((tablesToUpdate, tableName) => {
+    const table = schema.tables[tableName];
+    const columns = Object.keys(table).filter(columnName => table[columnName].nullable);
```

Comment:
To make this a bit more explicit, maybe the type fo `string` or `text` should be checked here as well?

Reply:
I was thinking this too - though we only update if the existing value is `''` so we shouldn't pick up any fields like that. Maybe it's better to be explicit tho :+1: